src/services/toast.service.js

```diff
@@ -0,0 +1,19 @@
+(function (angular) {
+	'use strict';
+
+	angular
+		.module('montage')
+		.factory('toast', toastService);
+
+	function toastService($mdToast) {
+		return {
+	  	simple
```

Comment:
Incorrect indentation level. Also, there is a mixture of spaces and tabs here. Stick to tabs.